runtime/binding-kafka/src/main/java/io/aklivity/zilla/runtime/binding/kafka/internal/stream/KafkaClientGroupFactory.java

```diff
@@ -362,7 +381,7 @@ public MessageConsumer newStream(
             {
                 final long resolvedId = resolved.id;
                 final List<KafkaServerConfig> servers = binding.servers();
-                final KafkaSaslConfig sasl = resolveSasl.apply(binding.sasl());
+                final KafkaSaslConfig sasl = resolveSaslSupplier.apply(0).apply(binding.sasl());
```

Comment:
Please define a constant `private static final int NO_REBALANCE_DELAY = 0` and use it here instead of `0`.